lib/widgets/lang_switch.dart

```diff
@@ -26,7 +24,9 @@ class LanguageTile extends StatelessWidget {
         );
         return ListTile(
           key: const Key('LanguageTile'),
-          contentPadding: EdgeInsets.zero,
+          contentPadding: EdgeInsets.symmetric(
+            horizontal: SizeConfig.blockSizeHorizontal! * 1.5,
```

Comment:
what's the purpose of the 1.5 here?

Reply:
removed the hard coded value 1.5 as it makes no big difference.